streaming-analytics/create.md

```diff
@@ -0,0 +1,4 @@
+# Creating an IBM Streaming Analytics instance
+1. Create an [IBM Streaming Analytics](https://console.bluemix.net/catalog/services/streaming-analytics) instance.
+![](streaming-analytics-catalog.png)
```

Comment:
Should probably put some alt text in here, even if it is just "Streaming Analytics"